scripts/entrypoint.sh

```diff
@@ -14,6 +15,12 @@ echo "Proceeding with JAVA_OPTIONS=$JAVA_OPTIONS"
 
 sed -i.bckp 's#host: .*#host: '$GREMLIN_HOST'#' ${GREMLIN_CONF}
 
+if grep -i 'graph.unique-instance-id=' "$PROPS" 1>/dev/null; then
```

Comment:
This will succeed for 

```
#graph.unique-instance-id=""
```
but with no effect..

add ^  ->   `grep -i '^graph.unique-instance-id='